OJP/OJP_Lines.xsd

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.vdv.de/ojp" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:siri="http://www.siri.org.uk/siri" targetNamespace="http://www.vdv.de/ojp" elementFormDefault="qualified" attributeFormDefault="unqualified" id="OJP_siri_service">
+	<xs:annotation>
+		<xs:documentation>OJP/OJP_Lines.xsd - OJP Line information request and response</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_all-v2.0.xsd"/>
+	<xs:include schemaLocation="OJP_All.xsd"/>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_modes-v1.1.xsd"/>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Service ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Response Group ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="LineInformationResponseGroup">
+		<xs:annotation>
+			<xs:documentation>Line Information response group.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="LineResult" type="LineResultStructure" minOccurs="0" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:group>
+	<xs:annotation>
+		<xs:documentation>=============================== LineInformation support functions ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:complexType name="LineResultStructure">
+		<xs:sequence>
+			<xs:element name="LineRef" type="siri:LineRefStructure">
+				<xs:annotation>
+					<xs:documentation>Reference to the LINE.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="siri:PublishedLineName" minOccurs="0"/>
+			<xs:element name="DirectionRef" type="siri:DirectionRefStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>DIRECTION of LINE if relevant for the shape.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Shape" type="siri:LineShapeStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The shape of the line. Normally a line has multiple routes. This is just the main one for presentation.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Probability" minOccurs="0">
```

Comment:
How do you suppose to get this probability? Does this refer to the route of the line, or the line itself?

Reply:
@herlitze  Should I name this a score? You do scores also for stops. I also would think this is the hidden ingredient for trip planners.

Reply:
@ue71603 There is only a LineDirectionFilterStructure in the request, so the probability refers to the route returned, correct?
Is the probability something like (number of trips of this line going this route)/(number of total trips of the line), or what would you expect here?
Is LineDirectionFilterStructure the correct request structure? What is an excluded line supposed to do?

Reply:
* A LineDirection can have different routes. Perhaps it is only one, perhaps there are several one. If you think a trip planner can say nothing, then we remove the probability. Frankly I have only a vague idea, if this is useful. I just copied things from the PlaceResults
* LineDirectionFilterStructure: Not that line. Again: Your expert judgement is needed, to say, if we need a new structure or not. I just reused things.

Reply:
Malte: Only ID then we don't need a score.
==> remove the PRobability.